Dockerfile.build

```diff
@@ -1,14 +1,14 @@
 # install protoc and protoc-gen-grpc-web
 FROM python:3.8 as protoc-build
 
-ENV PROTOC_ZIP=protoc-3.15.6-linux-x86_64.zip
-RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.15.6/$PROTOC_ZIP | bash \
+ENV PROTOC_ZIP=protoc-3.20.1-linux-x86_64.zip
```

Comment:
bumped these to the latest (working) versions available. matches the docs change at https://github.com/replicahq/model/pull/6103